CorsixTH/Lua/base_config.lua

```diff
@@ -133,19 +137,19 @@ local configuration = {
   },
 
   towns = {
-    {StartCash = 40000, InterestRate = 100}, -- Level 1
-    {StartCash = 40000, InterestRate = 200}, --  Level 2
-    {StartCash = 50000, InterestRate = 300}, --  Level 3
-    {StartCash = 50000, InterestRate = 400}, --  Level 4
-    {StartCash = 50000, InterestRate = 500}, --  Level 5
-    {StartCash = 50000, InterestRate = 600}, --  Level 6
-    {StartCash = 50000, InterestRate = 700}, --  Level 7
-    {StartCash = 60000, InterestRate = 700}, --  Level 8
-    {StartCash = 60000, InterestRate = 800}, --  Level 9
-    {StartCash = 60000, InterestRate = 800}, --  Level 10
-    {StartCash = 70000, InterestRate = 900}, --  Level 11
-    {StartCash = 70000, InterestRate = 900}, --  Level 12
-    {StartCash = 70000, InterestRate = 900}, --  Level 12
+    {StartCash = 40000, InterestRate = 100, StartRep = 500, OverdraftDiff = 200}, -- Level 1
```

Comment:
This line has 1 space in `-- Level` while the other have 2. As you're changing the line anyway, please correct that as well.

Reply:
That is, make it consistent, either by editing this line, or by editing all other lines :)